roles/package_verify_missing/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# vim: set ft=ansible:
+#
+# Verifies that a package is not installed
+#
+- name: Fail if rpm_name is undefined
+  fail:
+    msg: "rpm_name must be passed into this role/include"
+  when: rpm_name is undefined
+
+- name: Check for {{ rpm_name }} binary
+  command: command -v {{ rpm_name }}
+  register: c_output
+  failed_when: c_output.rc != 1
```

Comment:
It's the difference between "I expect this command to fail and return code 1" vs. just "I expect this command to fail". I.e. we don't care here what error code it actually returns, we just care that it fails. (In reality, those particular commands likely won't change their error code, but in the general case it's just more bulletproof to check for zero/non-zero).